### DISCUSSION TITLE
Color-changing reverse arrows on osu!2013

### DISCUSSION THREAD

**Discussion by quat1024** [ppy/osu#23138] [Ideas] [answered]
This is a parity feature request.
osu!stable's default skin will use black reverse arrows when the slider is a light color, which makes them stand out more:
![image](https://user-images.githubusercontent.com/2068021/230247161-4f5a1a84-762d-42b4-9348-66da1cfb2fa9.png)
(Picture from [the forums](https://osu.ppy.sh/community/forums/posts/2186820), i dont have stable installed atm)

But on lazer with the osu!2013 skin, reverse arrows are always white:
![image](https://user-images.githubusercontent.com/2068021/230246071-f4186b47-fa6d-430b-ad2b-d88cb38b983a.png)

I'd love to have black reverse arrows in lazer, maybe even as a proper skinnable. I know about the new "combo color normalization" setting, but IMO it's a bummer to make *everything* greyer when the osu!2013 hitcircleoverlay/numbers/slider track are readable with any color, really the only problematic element is the reverse arrow.

**Comment by peppy** [comment, marked as answer]
#23379
